src/js/angular/clustermanagement/controllers.js

```diff
@@ -241,7 +241,7 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $uibMod
                 })
                 .catch((error) => {
                     const failMessage = $translate.instant('cluster_management.delete_cluster_dialog.notifications.fail_delete');
-                    const failedNodesList = Object.keys(error.data)
+                    const failedNodesList = error.status === 403 ? getError(error.data, error.status) : Object.keys(error.data)
```

Comment:
The other actions in this view have this case handled. You can the same aproach here as well, so it is consistent.